FirebaseTestProject/ViewModels/AuthViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AuthViewModel.swift
+//  AuthViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+import SwiftUI
+
+@MainActor class AuthViewModel: ObservableObject {
+  
+  private let auth = Auth.auth()
+  
+  @AppStorage("userSignedIn") var userSignedIn: Bool = false
```

Comment:
Yes! `Auth` has an optional `currentUser` property that’s non-nil when there’s a user signed in. It’s also possible to [add a state change listener](https://firebase.google.com/docs/auth/ios/start#listen_for_authentication_state) that can keep that up to date. This might be preferable to storing a `userSignedIn` property in UserDefaults, since that way we’re working with fewer sources of truth.
I figured this out on a local branch. Would it help if I pushed that change to here?